requirements.txt

```diff
@@ -1,4 +1,10 @@
-
+# Django/Framework Packages
+django>=1.4.12,<=1.4.22
+django-model-utils==2.3.1
+South>=0.7.6
+djangorestframework>=2.3.5,<=2.3.14
+pytz>=2012h
+pycrypto>=2.6
```

Comment:
It was more pre-emptive, I wasn't hitting a conflict, but I just don't always want to have to keep updating my requirements when edx-platform does.

It seems like ripe for human error.